docs/feature/policies/global-policy.md

```diff
@@ -0,0 +1,133 @@
+          requires: # AND has the following Type references defined in the `spec.requires` property:
+            - path: "cap.type.gcp.auth.service-account"
+              # in any revision
+        inject:
+          typeInstances: # For such Implementation, inject the following TypeInstances: 
+            - id: 9038dcdc-e959-41c4-a690-d8ebf929ac0c
+              typeRef: # Find the alias of the Type reference in `spec.requires` property.
+                      # If it is defined, inject the TypeInstance with ID `9038dcdc-e959-41c4-a690-d8ebf929ac0c` under this alias.
+                path: "cap.type.gcp.auth.service-account"
+                revision: "0.1.0"
+              
+      - implementationConstraints: # In second place find and select Implementation which:
+          attributes: # contains the following attributes
+           - path: cap.attribute.cloud.provider.aws
+             # in any revision
+             
+      - implementationConstraints: # In third place, find and select Implementation which:
+          path: "cap.implementation.bitnami.postgresql.install" # has exact path
+          
+       # If not found any of such Implementations defined in `oneOf`, return error.
+    
+  - interface:
+      path: "cap.*" # For any other Interface
+      # (looked up in third place, if there is no entry under `rules` for a given Interface `path:revision` or `path`)
+    oneOf: # Engine follows the order of the Implementation selection,
+           # finishing when at least one matching Implementation is found
+      - implementationConstraints: # In first place, select Implementation which:
+          requires: # has the following Type references defined in the `spec.requires` property:
+            - path: "cap.core.type.platform.kubernetes"
+              # in any revision
+
+      - implementationConstraints: {} # If not found, fallback to any Implementation which has requirements that current system satisfies.
+
+      # If not found any of such Implementations defined in `oneOf`, return error. 
+```
+
+## Configuration
+
+You can view and update the Global policy using dedicated commands in Capact CLI. You can also modify policy during Capact installation or upgrade. This section describes all policy configuration options.
+
+### View current Global policy
+
+To view current Global policy rules, use the following command:
+
+```bash
+capact policy get
+```
+
+You can use additional flags to configure the command behavior, such as output format. Run `capact policy get -h` to see all available flags.
+
+### Modify Global policy using CLI
+
+> **NOTE:** If you update Policy using CLI, Policy rules will be restored to default every time you upgrade Capact installation. To avoid such scenario, update the Policy during Capact installation or upgrade. To learn how to do it, read the section [Modify Policy with Capact installation/upgrade overrides](#modify-policy-with-capact-installationupgrade-overrides).
```

Comment:
```suggestion
> **NOTE:** If you update Policy using CLI, Policy rules will be restored to default every time you upgrade Capact installation. To avoid such scenario, update the Policy during Capact installation or upgrade. To learn how to do it, read the section [Modify Policy with Capact installation/upgrade overrides](##modify-global-policy-with-capact-installationupgrade-overrides).
```